lib/foreman_scap_client/client.rb

```diff
@@ -9,10 +9,14 @@ module ForemanScapClient
   CONFIG_FILE = '/etc/foreman_scap_client/config.yaml'
 
   class Client
+    attr_reader :config, :policy_id, :tailored
+
     def run(policy_id)
       @policy_id = policy_id
       ensure_policy_exist
+      load_config
```

Comment:
getting following, because ensure_policy_exist on line above requires config to be present already

```
/usr/local/share/gems/gems/foreman_scap_client-0.3.0/lib/foreman_scap_client/client.rb:131:in `ensure_policy_exist': undefined method `[]' for nil:NilClass (NoMethodError)
        from /usr/local/share/gems/gems/foreman_scap_client-0.3.0/lib/foreman_scap_client/client.rb:16:in `run'
        from /usr/local/share/gems/gems/foreman_scap_client-0.3.0/bin/foreman_scap_client:10:in `<top (required)>'
        from /usr/bin/foreman_scap_client:23:in `load'
        from /usr/bin/foreman_scap_client:23:in `<main>'
```